frontend/src/components/InnerNavbar/InnerNavbar.scss

```diff
@@ -0,0 +1,9 @@
+
+
+.SideBar {
+  width: 240px;
+  height: 100%;
+  padding: 20px;
+  border-top-left-radius: 20px;
+  background-color: rgba($color: #090d2c, $alpha: 0.5);
```

Comment:
Could you please explain what this line is doing?

Reply:
As a general comment, I normally try to avoid transparent colours. This often tends to cause colour issues when you change colour scheme and instead requires more fiddling at that time. It's often best to identify the raw colour with 100% opacity, and use that number directly. 

We can store colour variables in a dedicated file at a later date so that we don't need to know the colour scheme but will have a mapping, so for now I would recommend copying the exact colour from the discord colour scheme and using that directly